lesson10/lesson10.js

```diff
@@ -0,0 +1,57 @@
+const ul = document.getElementById('js-list');
+const listItems = [
+  { to: "bookmark.html", img: "bookmark-icon.png", alt: "画像1", text: "ブックマーク" },
+  { to: "message.html", img: "mail-icon.png", alt: "画像2", text: "メッセージ" }
+];
+
+function showLoading() {
+  const loadingImg = document.createElement('img');
+  loadingImg.src = "loading-circle.gif";
+  loadingImg.id = "loading-img";
+  ul.appendChild(loadingImg);
+}
+
+function removeLoading() {
+  const loadingID = document.getElementById('loading-img');
+  loadingID.remove();
+}
+
+function createList(item) {
+  const fragment = document.createDocumentFragment();
+  for (let i = 0; i <= item.length - 1; i++) {
```

Comment:
```suggestion
  for (let i = 0; i < item.length;  i++) {
```
nits 
You can also write something like this

Reply:
Thank you for your comment! Thanks to you, we were able to simplify the inside of the function.https://github.com/rikokko/to_be_frontend/pull/18/commits/66a6e52a65ec7de0fda4452882ed9dae0eb6167a